### ISSUE TITLE
Amazon results in German

### ISSUE THREAD

**Issue by mgf897** [wassname/keywordshitter2#7]
Amazon results are in German.

in suggestion.js Amazon is hardcoded as 
`"http://completion.amazon.co.uk/search/complete?method=completion&search-alias=aps&mkt=4&callback=?&q=`" resulting in German responses. Please add a way to set `mkt=3` to allow English responses

**Comment by wassname** [comment]
Good idea, I welcome a PR. In the mean time we could set it to English since that's a better default than German.
